app/code/Magento/Wishlist/Model/Config.php

```diff
@@ -36,6 +38,11 @@ class Config
      */
     private $sharingEmailLimit;
 
+    /**
+     * @var int
+     */
+    protected $_sharignTextLimit;
```

Comment:
Please use `private $sharingTextLimit;`.  Avoid protected members at all costs

Reply:
That would be backward incompatible change, so no

Reply:
Ah I see it's dynamically declared field.  Should be public then?

Reply:
In general it should be private, but if we’ll do that - this variable will not be available in classes that extend this one. In order to make it available we decided to make them protected, so this variable will not be available outside of the class